frontend/routes/package/source.tsx

```diff
@@ -111,9 +115,9 @@ export default function PackagePage(
               : (
                 data.source.source.view
                   ? (
-                    <div class="ddoc border border-cyan-300 rounded">
+                    <div class="ddoc">
                       <div
-                        class="markdown children:!bg-transparent"
+                        class="markdown ddoc-full children:!bg-transparent"
```

Comment:
fixes a bug introduced by recent ddoc changes that were done for the docs site. This content didn't fill the container anymore like it should